src/main/java/org/prebid/server/bidder/improvedigital/proto/ImprovedigitalImpExt.java

```diff
@@ -0,0 +1,11 @@
+package org.prebid.server.bidder.improvedigital.proto;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
```

Comment:
This file is never used and can be deleted

Reply:
removed